.github/workflows/automated-updates-to-sam-cli.yml

```diff
@@ -0,0 +1,104 @@
+name: Update aws/aws-sam-cli with latest commit hash from aws/aws-sam-cli-app-templates and aws-sam-translator version
+
+on:
+  schedule:
+    - cron: "0 0/4 * * *" # run at the top of every 4 hour
+  workflow_dispatch: {}
+
+jobs:
+  updateCommitHash:
```

Comment:
can you change the job name to be more specific to the app template hash update.

Reply:
Can update, but this was how it was originally.